webapi/src/main/scala/org/knora/webapi/responders/v2/SearchResponderV2.scala

```diff
@@ -512,14 +521,19 @@ class SearchResponderV2(responderData: ResponderData) extends ResponderWithStand
           featureFactoryConfig = featureFactoryConfig
         )
 
-      // TODO: if the ORDER BY criterion is a property whose occurrence is not 1, then the logic does not work correctly
-      // TODO: the ORDER BY criterion has to be included in a GROUP BY statement, returning more than one row if property occurs more than once
+      // TODO-old: if the ORDER BY criterion is a property whose occurrence is not 1, then the logic does not work correctly
+      // TODO-old: the ORDER BY criterion has to be included in a GROUP BY statement, returning more than one row if property occurs more than once
```

Comment:
same as above

Reply:
IMO just leave it as it is w/o adding suffix.